provider/pkg/provider/helm_release.go

```diff
@@ -1337,6 +1343,31 @@ func excludeNulls(in interface{}) interface{} {
 	return in
 }
 
+func mapToInterface(in interface{}) interface{} {
+	switch reflect.TypeOf(in).Kind() {
+	case reflect.Map:
+		out := map[string]interface{}{}
+		m := in.(map[string]interface{})
+		for k, v := range m {
+			val := reflect.ValueOf(v)
+			if val.IsValid() {
+				out[k] = mapToInterface(v)
+			} else if v == nil {
+				out[k] = v
+			}
```

Comment:
```suggestion
			if val.IsValid() {
				if val.IsNil() {
					out[k] = nil
				} else {
					out[k] = mapToInterface(v)
				}
			}
```

Reply:
@lblackstone Oh great it looks like we ended up in similar places. I'll commit yours as I am certain it is more idiomatic. Thank you!